app/controllers/preprints/provider/moderators.js

```diff
@@ -81,10 +83,15 @@ export default Controller.extend(Analytics, moderatorsQueryParams.Mixin, {
     }),
 
     deleteModerator: task(function* (moderatorInstance) {
+        const removingSelf = moderatorInstance.id === this.get('currentUser.user.id');
         try {
             yield moderatorInstance.destroyRecord({ adapterOptions: { provider: this.get('theme.provider.id') } });
             moderatorInstance.unloadRecord(); // https://github.com/emberjs/data/issues/5014
 
+            if (removingSelf) {
+                return;
```

Comment:
this just `returns` but the others return `true` or `false`... but it kinda looks like the `true` and `false` might be unused?